### PR TITLE
Fix widget tests of quality metrics 

### PR DESCRIPTION
**Pull request by h-mayorquin**
The widgets tests are failing on master. I think I know where the bug is coming from. Testing here. 

Also, looking at the setup of the widgets tests, it seems that they depend heavily on post-processing and quality metrics. Changes on those modules should trigger the widgets tests. Adding that here to the CI.

### CODE REVIEW

**Comment by alejoe91** [comment]
> The widgets tests are failing on master. I think I know where the bug is coming from. Testing here.
> 
> Also, looking at the setup of the widgets tests, it seems that they depend heavily on post-processing and quality metrics. Changes on those modules should trigger the widgets tests. Adding that here to the CI.

Gooood idea!

File: spikeinterface/qualitymetrics/quality_metric_list.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "firing_rate" : compute_firing_rates,
     "presence_ratio" : compute_presence_ratios,
     "snr" : compute_snrs,
-    "isi_violation" : compute_isi_violations,
+    "isi_violations" : compute_isi_violations,
```
Comment (alejoe91) [inline spikeinterface/qualitymetrics/quality_metric_list.py:35]:
```suggestion
    "isi_violation" : compute_isi_violations,
```

Can you change instead this line: https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/widgets/tests/test_widgets.py#L51 ?

Names are more consistent now, i.e., QM names are always singular (while functions are plural)

**Review by alejoe91 — CHANGES_REQUESTED** [review]
Please see comment ;)

File: spikeinterface/qualitymetrics/quality_metric_list.py (hunk 1 above)
Comment (h-mayorquin) [reply spikeinterface/qualitymetrics/quality_metric_list.py]: Done.

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**